spec/parliament/ntriple/utils_spec.rb

```
@@ -184,4 +184,24 @@
       expect(sorted_incumbencies[1].end_date).to eq(DateTime.new(2015, 3, 30))
     end
   end
+
+  describe '#multi_direction_sort' do
+    it 'returns a response sorted by member_count (desc) and name (asc)' do
```

We need to add tests for (asc), (asc) - (desc), (desc) + what if you provide more that two?

---

Done - there are now tests for asc asc, desc desc, desc asc and three parameters.

---

Just to check - is there an upper limit to the number of sorts and orders you will allow?

If so we should raise an error if more than that - if not we should document and maybe have like a 10 deep test for sanity checking

---

Done - there is now an extra test for 5 sort parameters